### ISSUE TITLE
Prepare scripts for usage with any database and language

### ISSUE THREAD

**Issue by GoogleCodeExporter** [rowiki/wikiro#17]
```
All scripts should be usable for any database and language that have a config.


```

Original issue reported on code.google.com by `strain...@gmail.com` on 10 Feb 2015 at 11:30
- Blocked on: #18, #19, #20, #21, #22, #23, #25, #26, #28, #29

**Comment by strainu** [comment]
The code can now run on at least 3 different databases (LMIRO, RANRO, WLEMD). Getting rid of references to RAN is nice to have, but not longer compulsory.
